client/components/mma/accountoverview/PersonalisedHeader.tsx

```diff
@@ -1,12 +1,26 @@
 import { css } from '@emotion/react';
 import { headline, space } from '@guardian/source-foundations';
+import { dateString } from '../../../../shared/dates';
 import type { MembersDataApiResponse } from '../../../../shared/productResponse';
 import { isProduct, sortByJoinDate } from '../../../../shared/productResponse';
 
 interface PersonalisedHeaderProps {
 	mdapiResponse: MembersDataApiResponse;
 }
 
+function calculateTimeOfDay() {
+	const currentHour = new Date().getHours();
+
+	if (currentHour < 12) {
+		return 'Good morning';
+	}
+	if (currentHour < 18) {
+		return 'Good afternoon';
+	} else {
```

Comment:
I'm pretty sure you pointed this out before, but you don't really need an `else` here.